### ISSUE TITLE
Only be one argument

### ISSUE THREAD

**Issue by seel-channel** [taodo2291/xgesture_flutter#1]
Please, let arguments callbacks only be one. 

I use onMoveStart, onMoveUpdate and onMoveEnd. Each needs 3 arguments (int, Offset, Offset), can you create a class contains the 3 arguments?  For example MoveDetails 

FROM :
```dart
Offset position = Offset.zero;

      XGestureDetector(
        child: child,
        onMoveEnd: (_, __, ___) => setState(() => isDragging = false),
        onMoveStart: (_, __, ___) => setState(() => isDragging = true),
        onMoveUpdate: (_, __, ___, delta) {
          setState(() {
            position = Offset(
              position.dx + delta.dx,
              position.dy + delta.dy,
            );
          });
        },
      ),
```

TO :
```dart
Offset position = Offset.zero;

      XGestureDetector(
        child: child,
        onMoveEnd: (_) => setState(() => isDragging = false),
        onMoveStart: (_) => setState(() => isDragging = true),
        onMoveUpdate: (MoveDetails details) {
          setState(() {
            position = Offset(
              position.dx + details.delta.dx,
              position.dy + details.delta.dy,
            );
          });
        },
      ),
```

**Comment by taodo2291** [comment]
Thanks for your suggestion. 

I'll update it soon.

**Comment by taodo2291** [comment]
resolved! The new changes included in version 0.0.5
[pub - gesture_x_detector](https://pub.dev/packages/gesture_x_detector/versions/0.0.5)
